src/js/common/components/Modal.jsx

```diff
@@ -49,6 +49,10 @@ class Modal extends React.Component {
     }
   }
 
+  componentWillUnmount() {
+    document.removeEventListener('focus', this.state.focusListener, true);
```

Comment:
Unbinds the event listener when the component unmounts. I think this is an issue when a modal renders and then no longer renders in an update of its parent component. I was getting some type errors about the modal being null when I made a parent component of a modal update without containing it.

Tagging @jbalboni for awareness.